File: Backend/Explorify.Api.Plubications/Explorify.Api.Publications.Domain/Entities/Publication.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;
using System;

namespace Explorify.Api.Publications.Domain.Entities
{
    public class Publication
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public string Id { get; set; } = string.Empty;

        [BsonElement("imageUrl")]
        public string ImageUrl { get; set; } = string.Empty;

        [BsonElement("title")]
        public string Title { get; set; } = string.Empty;

        [BsonElement("description")]
        public string Description { get; set; } = string.Empty;

        [BsonElement("location")]
        public string Location { get; set; } = string.Empty;

        [BsonElement("userId")]
        public string UserId { get; set; } = string.Empty;

        [BsonElement("createdAt")]
        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
    }
}

File: ExplorifyApp/app/src/main/java/com/example/explorifyapp/data/remote/model/Publication.kt
package com.example.explorifyapp.data.remote.model

data class Publication(
    val id: String,
    val imageUrl: String,
    val title: String,
    val description: String,
    val location: String,
    val latitud: String? = null,
    val longitud: String? = null,
    val userId: String,
    val createdAt: String
)